docs/docs/40-contributor-guide/10-hacking-on-kargo.md

```diff
@@ -455,13 +455,11 @@ make hack-serve-docs
 ```
 
 :::info
-If you wish to opt-out of executing code-generation within a container (for
-performance reasons, perhaps), drop the `hack-` prefix from the target to run the docs natively on your system:
+If you want to build and serve the docs on your local machine, run the following command:
 
 ```shell
 make serve-docs
 ```
 
-This will require quite a variety of tools to be installed locally, so we do not
-recommend this if you can avoid it.
+This will require you to install the [`pnpm`](https://pnpm.io/installation) tool locally on your machine.
```

Comment:
I do not think this is true? `hack-build-dev-tools` should run within a container, and not touch your local system.

Reply:
There is another tool. It's Node. If you run in a container, you get the version of Node we use instead of whatever happens to be on your system, which is why we discourage that.

Reply:
`serve-docs` requires you to install `pnpm` here. If a user would like to go through building the docs locally, he must perform steps that requires himto install the tool (ofc, prerequisites to install the tool is required but that's something the official docs of pnpm would cover).